### ISSUE TITLE
Escaped Strings in Quick Values improperly "Add to Search Query"

### ISSUE THREAD

**Issue by NickMeves** [Graylog2/graylog2-web-interface#1484]
Thanks for the quick fix on Issue #1455 -- alleviates alot of security concerns.

Related to that issue though:

Any values clicked on from Quick Values will have the HTML Escaped version in the search bar (&amp;lt; &amp;gt; in all their glory) which doesn't find the appropriate values in the prefix search context on the field.

**Comment by edmundoa** [comment]
Hi,

How did you manage to get `<` and `>` into the quick values? I try to reproduce it last time, but in the end I fed one of those values by hand into the Javascript console. It would be great if you could share one of those messages (removing any sensitive information of course), so we can test it properly :)

**Comment by NickMeves** [comment]
When testing I just did it the quick route and added a static field with HTML markup onto a random HTTP input on the fly.

I also overrided the source and replaced source with HTML markup to make sure the graph on the sources page was escaped too (it is :) )

In production we have email logs (message-ids in emails frequently are wrapped in &lt; &gt; ) and we have HTML links.  Just sending those over via GELF TCP got the &lt; &gt; into the system for us.
